pkg/northbound/gnmi/subscribe_test.go

```diff
@@ -321,3 +255,94 @@ func Test_ErrorDoubleSubscription(t *testing.T) {
 	err = server.Subscribe(serverFake)
 	assert.ErrorContains(t, err, "is already registered")
 }
+
+func Test_Poll(t *testing.T) {
+	server, _ := setUp()
+
+	path, err := utils.ParseGNMIElements([]string{"test1:cont1a", "cont2a", "leaf2a"})
+
+	assert.NilError(t, err, "Unexpected error doing parsing")
+
+	path.Target = "Device1"
+
+	request := buildRequest(path, gnmi.SubscriptionList_POLL)
+
+	pollrequest := &gnmi.SubscribeRequest{
+		Request: &gnmi.SubscribeRequest_Poll{
+			Poll: &gnmi.Poll{},
+		},
+	}
+
+	responsesChan := make(chan *gnmi.SubscribeResponse, 1)
+	serverFake := gNMISubscribeServerPollFake{
+		Request:     request,
+		Responses:   responsesChan,
+		Signal:      make(chan struct{}),
+		first:       true,
+		PollRequest: pollrequest,
+	}
+
+	//TODO need to block
+	go func() {
+		err = server.Subscribe(serverFake)
```

Comment:
again - can be a 1 liner:
go server.Subscribe(serverFake)

Reply:
not really, otherwise the erro variable is never set